hx_lti_initializer/utils.py

```diff
@@ -128,6 +128,8 @@ def save_session(request, **kwargs):
         "roles": ["hx_roles", []],
         "is_staff": ["is_staff", False],
         "is_instructor": ["is_instructor", False],
+        "is_graded": ['is_graded', False],
```

Comment:
@arthurian I thought adding this set the default is_graded to false. Shouldn't this get loaded every time save_sessions is called? or only when you specifically set it one way or the other?